### ISSUE TITLE
nginx delayed start

### ISSUE THREAD

**Issue by alfs** [MONROE-PROJECT/Maintenance#135]
At node reboot, nginx fails to start because it cannot bind 172.17.0.1:

```
● nginx.service - A high performance web server and a reverse proxy server
   Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Thu 2018-10-11 11:43:01 UTC; 1min 29s ago
     Docs: man:nginx(8)
  Process: 753 ExecStartPre=/usr/sbin/nginx -t -q -g daemon on; master_process on; (code=exited, status=1/FAILURE)

Oct 11 11:43:00 Monroe000db940d8c4 systemd[1]: Starting A high performance web server and a reverse proxy server...
Oct 11 11:43:01 Monroe000db940d8c4 nginx[753]: nginx: [emerg] bind() to 172.17.0.1:80 failed (99: Cannot assign requested address)
```

It's because that interface (docker0) is not available when the nginx service starts
Biteback will eventually correct this, but it causes the proxied services at port 88 (/dlb, /modems, etc) to malfunction until biteback has executed.

Suggested fix:

Change /lib/systemd/system/nginx.service 
```
-After=network.target
+After=docker.service
```

@relet, if you add this to ansible I can test that it works. Not urgent, but an improvement for nodes that often reboot (i.e. vehicular mounted)

**Comment by alfs** [comment]
Tested the above manually + rebooting. Works as expected.

**Comment by relet** [comment]
Can you double check that the fix is working as intended? We are not
actually using the docker0 interface in MONROE, iirc it is disabled in the
docker configuration (once fully configured).

**Comment by alfs** [comment]
I worked on a node (232) that I thought was fully configured (ansible completed, several reboots); but on other nodes I see there is no docker0 interface. Instead the 172 address is on the metadata interface.

Basing the systemd service on docker.services was on the assumption that docker configured the interface, but depending on whatever creates the metadata interface is then better. Perhaps metadata-exporter.service?

**Comment by relet** [comment]
No, it is the monroe-experiments script. I don't think we can catch the
correct boot order from systemd.
